src-ui/lib/state/Books.spec.ts

```diff
@@ -47,27 +47,24 @@ describe('books', () => {
   })
 
   describe('with several books preloaded', () => {
+    let duneMessiahId: string
+
     beforeEach(() => {
       books.add(duneMessiah)
       books.add(princessAndGrilledCheese)
+      duneMessiahId = books.value[0].id
     })
 
     it('should contain two books after adding', () => {
       expect(books.value.length).toBe(2)
     })
 
     describe('#remove', () => {
-      it('should remove a book by ISBN-10', () => {
-        books.remove(duneMessiah.isbn10 as string)
+      it('should remove a book by ID', () => {
+        books.remove(duneMessiahId)
         expect(books.value.length).toBe(1)
         expect(books.value[0].title).toBe(princessAndGrilledCheese.title)
       })
-
-      it('should remove a book by ISBN-13', () => {
-        books.remove(princessAndGrilledCheese.isbn13 as string)
-        expect(books.value.length).toBe(1)
-        expect(books.value[0].title).toBe('Dune Messiah')
-      })
```

Comment:
What do you think of providing a method to either find or remove by `isbn`? At some point, I may want to donate books; and it would be useful to have an API that supported some workflow for "I have a scanned book, and I would like to remove it from the db".